build/parser.js

```diff
@@ -508,10 +508,18 @@ class Parser {
 
     // Sort by drop rate
     if (item.drops) {
-      item.drops.sort((a, b) => b.chance - a.chance)
+      item.drops.sort((a, b) => this.comparator(a, b));
```

Comment:
use the function directly as the parameter for sort instead of directing the lambda to it extra, like....
```suggestion
      item.drops.sort(this.comparator);
```